### PR TITLE
PEM 8.6 - Release branch

### PR DESCRIPTION
**Pull request by nidhibhammar**
Applied the changes to the docs as per https://github.com/EnterpriseDB/pem/pull/105 from rst files

### CODE REVIEW

File: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/20_monitoring_a_pgd_cluster.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Monitoring a PGD Group"
+---
+
+<div id="monitoring_a_pgd_cluster" class="registered_link"></div>
+
+PGD (Bi-Directional Replication) is a PostgreSQL extension that provides a multi-master replicaton between the Postgres clusters. Before you monitor PGD through PEM console, you must ensure that your PGD nodes are up and running. For more information about PGD please visit the [EDB Doc Website](https://www.enterprisedb.com/docs/pgd/latest/).
```
Comment (ebgitelman) [inline product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/20_monitoring_a_pgd_cluster.mdx:7]: Does PGD stand for Bi-Directional Replication?? In any case, usually the full name is first and then the abbreviation.

File: product_docs/docs/pem/8/monitoring_BDR_nodes.mdx [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+title: "Monitoring PGD"
+# This file is moved for pem_ent_feat/17_monitoring_PGD_nodes.mdx
 ---
 
 !!! Tip "New Feature "
-    BDR support is available in PEM version 8.1.0 and later.
+    PGD support is available in PEM version 8.1.0 and later.
 
-BDR (Bi-Directional Replication) is a PostgreSQL extension that provides multi-master replication between the Postgres clusters. Before you monitor BDR through PEM console, you must ensure that your BDR nodes are up and running. For more information, see [BDR](/bdr/latest).
+PGD (Bi-Directional Replication) is a PostgreSQL extension that provides multimaster replication between the Postgres clusters. Before you monitor PGD through PEM console, you must ensure that your PGD nodes are up and running. For more information, see [PGD](/pgd/latest).
```
Comment (ebgitelman) [inline product_docs/docs/pem/8/monitoring_BDR_nodes.mdx:9]: Is this what PGD stands for? I think you want the full name written out first and then the initialism.

File: product_docs/docs/pem/8/pem_rel_notes/08_810_rel_notes.mdx [hunk 3]
```diff
@@ -6,12 +6,12 @@ New features, enhancements, bug fixes, and other changes in PEM 8.1.0 include:
...
-| New Feature | Kerberos security and authentication for PEM Web Client through GSSAPI or Active Directory Domain Services. Prevent password interception and prove identity required by highly regulated industries. [Support Ticket #1044678] See [Configuring the PEM Server to use Kerberos Authentication](../installing_pem_server/authentication_options/configuring_the_pem_server_to_use_kerberos_authentication/). | PEM-3559                                             |
+| New Feature | PGD (Bi-Directional Replication) dashboards and probes to monitor status and activities for admin, nodes, and groups. For example, if replication starts falling behind, the administrator is alerted and can take proactive action. See [Monitoring PGD nodes](../monitoring_PGD_nodes/).                                                                                                                   | PEM-3893                                             |
```
Comment (ebgitelman) [inline product_docs/docs/pem/8/pem_rel_notes/08_810_rel_notes.mdx:9]: Is it correct that PGD stands for Bi-Directional Replication?

File: product_docs/docs/pem/8/pem_rel_notes/05_830_rel_notes.mdx [hunk 4]
```diff
@@ -11,7 +11,7 @@ New features, enhancements, bug fixes, and other changes in PEM 8.3.0 include:
 | Enhancement | Support Export/Import Custom charts and Custom Dashboards using PEM GUI and RESTapi.                                                                                | PEM-4300, PEM-4325, PEM-3443, PEM-4215               |
 | Enhancement | Mask the username from the login related errors when displaying on GUI and return generic error message. [Support Ticket #1341245]                                  | PEM-4150                                             |
 | Enhancement | Support for PostgreSQL, EDB Postgres Advanced Server and EDB Postgres Extended Server version 14 is added.                                                                 | PEM-4091                                             |
-| Enhancement | Support for monitoring status and activities for bdr.stat_subscription and bdr.stat_relation on BDR (Bi-Directional Replication) dashboards and probes is added.    | PEM-4179                                             |
+| Enhancement | Support for monitoring status and activities for pgd.stat_subscription and pgd.stat_relation on PGD (Bi-Directional Replication) dashboards and probes is added.    | PEM-4179                                             |
```
Comment (ebgitelman) [inline product_docs/docs/pem/8/pem_rel_notes/05_830_rel_notes.mdx:14]: PGD stands for Bi-Directional Replication?

**Review by ebgitelman — COMMENTED** [review]
I didn't review the online help files because these have not been edited at all so any edits I make would be inconsistent with the rest of the online help.

**Comment by drothery-edb** [comment]
@nidhibhammar I assume you want the Monitoring EDB Postgres Distributed topic where the Monitoring BDR one was, but it is appearing at the bottom of the nav.

File: product_docs/docs/pem/8/pem_rel_notes/02_860_rel_notes.mdx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Version 8.6.0"
+---
+
+New features, enhancements, bug fixes, and other changes in PEM 8.6.0 include:
+
+| Type         | Description                                                                                                                                                                                                                                                          | ID        |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| --------- |
+| Enhancement  | Allow PEM agent connections to use SSL passwords for greater interoperability with multi-domain SSL certificates and compliance satisfaction [Support Tickets #73628, #82574, #82144, #82175]                                                                        | PEM-4276  |
```
[the remark this answers is not in the document]
Comment (nidhibhammar) [reply product_docs/docs/pem/8/pem_rel_notes/02_860_rel_notes.mdx]: Done changes in separate commit.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**